package.json

```diff
@@ -27,8 +27,10 @@
         "redux-watch": "1.1.1",
         "semantic-ui-react": "0.77.2",
         "stringifier": "1.3.0",
-        "styled-components": "3.2.6",
+        "styled-components": "^3.2.6",
```

Comment:
could you explain the caret here?

Reply:
i did not see that.
i think it probably happened when i installed styled-components-grid first through npm (which broke the system as yarn.lock did not get updated) and then installed it with yarn as well which made the system work again.

what it means (according to the internet) is that it will match any 3.x.x release including but will hold off on 4.0.0 (it allows patch and minor updates for versions).